clang/test/Driver/hlsl-lang-targets-spirv.hlsl

```diff
@@ -2,8 +2,8 @@
 
 // Supported targets
 //
-// RUN: %clang -target dxil-unknown-shadermodel6.2-pixel %s -S -o /dev/null 2>&1 | FileCheck --check-prefix=CHECK-VALID %s
-// RUN: %clang -target spirv-unknown-shadermodel6.2-library %s -S -o /dev/null 2>&1 | FileCheck --check-prefix=CHECK-VALID %s
+// RUN: %clang -target dxil-unknown-shadermodel6.2-compute %s -S -o /dev/null 2>&1 | FileCheck --allow-empty --check-prefix=CHECK-VALID %s
+// RUN: %clang -target spirv-unknown-shadermodel6.2-compute %s -S -o /dev/null 2>&1 | FileCheck --allow-empty --check-prefix=CHECK-VALID %s
```

Comment:
I’m a little confused by this test change. It seems to me that the unmodified test should still have been valid. Did something change here?

Reply:
This test is actually failing at tip of tree, probably due to a change on our side where we started explicitly failing on not-yet-supported shader types, but I don't think we knew about this test or made a habit of running it so it just fell through the cracks. I can split it out into a separate patch if you'd prefer though.

Reply:
Oh, to clarify, the `dxil` one was passing, but it seemed a little odd to leave it as `pixel` when I made the shader `compute` to satisfy `spirv`'s current capabilities. `dxil` also doesn't actually need the `--allow-empty` flag because it spits out a warning, but since that's not what this test should be testing, I thought it made sense to apply to both.

Reply:
Separated the test fix into: https://github.com/llvm/llvm-project/pull/70330